base/Dockerfile.ubuntu_20.04.gnu

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:20.04
+RUN apt-get update -y && \
+   DEBIAN_FRONTEND=noninteractive \
+   apt-get install -y --no-install-recommends \
+           autoconf \
+           automake \
+           cmake \
+           curl \
+           g++ \
+           gcc \
+           gfortran \
+           git \
+           libcurl4-openssl-dev \
+           libmpich-dev \
+           libtool \
+           libexpat1-dev \
+           make \
+           pkg-config \
+           vim \
+           wget \
+           unzip \
+           python3-dev \
+           python3-pip
```

Comment:
we need tar, gzip, doxygen as well.

Reply:
tar and gzip are available without separately installing. doxygen takes up about 160MB of space; can that be optional if a user need it?

Reply:
I see. Ok. we can leave doxygen for now.